### ISSUE TITLE
thin-plate-spline question

### ISSUE THREAD

**Issue by cheind** [hkchengrex/Mask-Propagation#37]
Hey,

I'm the author of https://github.com/cheind/py-thin-plate-spline and discovered your dependency here. I've tracked its usage down to https://github.com/hkchengrex/Mask-Propagation/blob/b5d8e61c87e1944426c3aed43685fd0eb336353a/dataset/tps.py#L16

I'm curious, what are you using it for? It seems like you are augmenting training data by warping the data with thin-plate-spline model. Was the library useful in this respect, i.e did it improve performance (quantifyable)? I'd like add a linkto your paper/model the project page if you don't mind?

**Comment by hkchengrex** [comment]
Hi, 

Thank you for your code. I am using it for data augmentation.
I have not specifically checked the quantitative improvement from this augmentation but the generated results do look promising. I liked how it is able to augment the image beyond standard affine/perspective transformation.  
TPS was used in a previous paper: https://arxiv.org/pdf/1612.02646.pdf so I just followed from there.

I have also used it in my other paper: https://github.com/hkchengrex/STCN for the same purpose.

You are welcomed to add links.
